modules/security-context-constraints-psa-coexistence.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-context-constraints-psa-coexistence_{context}"]
+= Pod security admission and security context constraints
+
+Pod security admission standards and security context constraints are reconciled and enforced by two independent controllers.   The two controllers work together using the following processes to enforce security policies and preserve backward compatibility:
```

Comment:
```suggestion
Pod security admission standards and security context constraints are reconciled and enforced by two independent controllers. The two controllers work independently using the following processes to enforce security policies:
```
(neither of the controllers described contributes to backward compatibility)

Reply:
fixed